### PR TITLE
no changes

### PR DESCRIPTION
**Pull request by mujeeb100**

### CODE REVIEW

**Review by syedabdullahali — APPROVED** [review]

**Comment by Rehan-Khan-Engineer** [comment]
Can you add a description and a live link of PR! @mujeeb100

File: style.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+.wheather h2 {
+display: flex;
+    align-items: center;
+  margin-bottom: 0;
+}
+
+/* .weather img{
+    transform:scale(2);
+} */
```
Comment (Rehan-Khan-Engineer) [inline style.css]: Could you explain this?

File: script.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+"use strict";
+
```
Comment (Rehan-Khan-Engineer) [inline script.js:2]: Could you explain why strict mode is required here? @mujeeb100

Comment (mujeeb100) [reply script.js:2]: code should be executed in the strict mode,
if you are not declaring the variable in the function it allows you to error
mistyping a variable name creates a new global variable and all others things,
syntax error

File: style.css (hunk 1 above)
Comment (mujeeb100) [reply style.css]: it is comment ??

Comment (Rehan-Khan-Engineer) [reply style.css]: If not needed can you remove it? @mujeeb100

Comment (mujeeb100) [reply style.css]: yeah sure brother

**Review by zaidansari42 — APPROVED** [review]

**Comment by mujeeb100** [comment]
successful projects

**Pull request closed without merging**